packages/react/src/accordion/root/AccordionRoot.tsx

```diff
@@ -162,6 +205,7 @@ AccordionRoot.propTypes /* remove-proptypes */ = {
   /**
    * Callback fired when an Accordion section is opened or closed.
    * The value representing the involved section is provided as an argument.
+   * @default NOOP
```

Comment:
The `generateProptypes` script will complain otherwise since hook params are now mandatory and we must set a default value in the component, we'd have to modify that script

Reply:
Can we set it to undefined so that it makes sense among other prop types?

<img width="566" alt="image" src="https://github.com/user-attachments/assets/5a6eed3c-e67d-4206-8b3d-c5f22f1f30c3">

Reply:
I see other handlers, like Dialog's onOpenChange, don't have a default—will that change as well? I don't have the context about the prop types script

Reply:
Actually it's not just about the proptypes, I wonder if we need to make an exception for optional event handlers

Reply:
OK fixed, just needed to set the default elsewhere